onnxruntime/contrib_ops/rocm/bert/util.cc

```diff
@@ -27,7 +27,7 @@ void Timer::End() {
   HIP_CHECK(hipEventSynchronize(end_));
 }
 
-float Timer::time() {
+double Timer::Duration() {
   float time;
```

Comment:
What is the reason to use double? Consider adding a note?

If we use double, shall we have "return double(time);" to indicate this conversion is intentional?